### DISCUSSION TITLE
Export flow

### DISCUSSION THREAD

**Discussion by LeenaAlrajhi** [postmanlabs/postman-flows#144] [Q&A]
Hello postman team,
I would ask if Is there any planning to provide an export feature in the flow with JSON format? In order to facilitate the transfer of it from one workspace to another and share it with others.

If yes, when will release it?

Thank you in advance

**Comment by saswatds** [comment]
Hey @LeenaAlrajhi, Check out this discussion #35 where I have tried answering this specific question.
